pages/blog/_posts/2020-03-30-catch-imsi-from-vowifi-session.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blogpost
+permalink: /blog/vowifi-imsi-leak/
+title: VoWifi leaking IMSI
+date: 2020-03-30
+tags: telco 4g epdg vowifi networking security
+author: <a href="https://www.linkedin.com/in/rskjetlein/">Roger Skjetlein</a>
+---
+
+4g offers more services than the earlier generation such as 3g and 2g. One of
+the service that really have gained traction later years is VoLTE, that is Voice
+over LTE and later that we will go more in dept regarding security is VoWifi,
+Voice over Wifi.
+
+VoWifi is beneficial in terms of using any Wifi connection offering public
+internet access thus extending and improving the coverage and connectivity.
+Think of it in similar ways of building our own eNodeB cellular network, but
+using commodity wifi components instead and avoiding the strict regulation and
+licensing spectrum.
+
+## What is IMSI
+*The international mobile subscriber identity (IMSI) is a number that uniquely
+identifies every user of a cellular network.[1] It is stored as a 64-bit field
+and is sent by the mobile device to the network. It is also used for acquiring
+other details of the mobile in the home location register (HLR) or as locally
+copied in the visitor location register. To prevent eavesdroppers from
+identifying and tracking the subscriber on the radio interface, the IMSI is sent
+as rarely as possible and a randomly-generated TMSI is sent instead.*
```

Comment:
```suggestion
> The international mobile subscriber identity (IMSI) is a number that uniquely
> identifies every user of a cellular network.[1] It is stored as a 64-bit field
> and is sent by the mobile device to the network. It is also used for acquiring
> other details of the mobile in the home location register (HLR) or as locally
> copied in the visitor location register. To prevent eavesdroppers from
> identifying and tracking the subscriber on the radio interface, the IMSI is sent
> as rarely as possible and a randomly-generated TMSI is sent instead.
```

Reply:
Should probably remove the `[1]`.

Reply:
Fixed